plugins/database/src/general_query.cpp

```diff
@@ -2330,6 +2331,8 @@ int chl_gen_query_access_control_setup_impl(
     static int recursiveCall = 0;
 #endif
 
+    // TODO: do I keep these and others like it?
+    // note: similar messages in the other files were "low-value"
     if ( logSQLGenQuery ) {
```

Comment:
i think it can go away

Reply:
I agree. Delete those and `logSQLGenQuery` if possible.